lib/Importer.js

```diff
@@ -452,4 +435,25 @@ export default class Importer {
     }
     return messageParts.join(' ');
   }
+
+  injectSideEffectImports(importStatements: ImportStatements): Array<string> {
+    const addedImports = [];
```

Comment:
If I remember correctly, it was because I couldn't find a good way to merge two sets without first turning at least one of them into an array.